code/modules/mob/living/simple_animal/hostile/statue.dm

```diff
@@ -211,7 +211,7 @@
 
 		else
 			if(target.see_invisible == SEE_INVISIBLE_LIVING)
-				target.see_invisible = SEE_INVISIBLE_OBSERVER_NOLIGHTING
+				target.see_invisible = SEE_INVISIBLE_NOLIGHTING
 				name = "Toggle Nightvision \[ON]"
 			else
 				target.see_invisible = SEE_INVISIBLE_LIVING
```

Comment:
Here are comments about the files changed below:
- some vars removed in mob_defines.dm
- Mob/Login() no longer deletes hud_used and creates a new one, that's stupid. It now calls create_mob_hud if there is none, otherwise it calls show_hud() to update the hud. It also calls reset_perspective()